project/Assembly.scala

```diff
@@ -12,12 +12,14 @@ object Assembly {
       project
         .settings(
           mainClass in assembly := mainClass.value,
-          assemblyOutputPath in assembly := file(s"/tmp/${name.value}.jar"),
+          assemblyOutputPath in assembly := file(
+                s"${(baseDirectory in ThisBuild).value}/dist/${name.value}-${version.value}.jar"
```

Comment:
That is funny, because I would expect the same behavior for below paths:
- `s"${(baseDirectory in ThisBuild).value}/dist/${name.value}-${version.value}.jar"`
- `s"./dist/${name.value}-${version.value}.jar"`

but the first one creates files with version: `...-0.2020.0022` while in the same time the latter creates: `...-0.2020.0022-SNAPSHOT`.

Which one is correct, because I've started thinking that we should build it with `-SNAPSHOT`.

Reply:
Interesting, but it might depend on where the .git structure is? The snapshot suffix depends on changes seen by git wrt the clean commit status